src/dmd/expressionsem.d

```diff
@@ -5019,11 +5046,14 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
         sc2.tinst = null;
         sc2.minst = null;
         sc2.flags |= SCOPE.fullinst;
-        Type t = e.targ.trySemantic(e.loc, sc2);
+        Type t = null;
+        if (e.tok2 == TOK.package_ || e.tok2 == TOK.module_) // These is() expressions are special because they can work on modules, not just types, so we'll skip the semantic check for them.
+            t = e.targ;
```

Comment:
they might be aliases to package/module symbols

Reply:
Now that I've actually gone and read the wiki, is this an appropriate use of the `Type.resolve` method that's mentioned in the [DMD Source Guide](https://wiki.dlang.org/DMD_Source_Guide#Determine_if_a_DMD_.27Type.27_is_an_actual_type.2C_expression.2C_or_a_symbol)?